packages/expo/src/executors/download/download.impl.ts

```diff
@@ -28,6 +28,7 @@ export interface ReactNativeDownloadOutput {
 const streamPipeline = promisify(pipeline);
 
 /**
+ * @deprecated this executor is to be removed in NX 17. It is no longer used.
```

Comment:
Add a `TODO(v17)` so we can find it later.

Reply:
added